File: src/frontend/tests/fixtures/apphost-languages/csharp/apphost.cs
#:sdk Aspire.AppHost.Sdk@13.6.0-preview.1.26458.6
#:property AspireUseCliBundle=true

var builder = DistributedApplication.CreateBuilder(args);

builder.AddExecutable("web", "node", ".", "service.mjs")
    .WithHttpEndpoint(env: "PORT")
    .WithExternalHttpEndpoints();

builder.Build().Run();

File: src/frontend/tests/fixtures/apphost-languages/java/AppHost.java
import aspire.*;

void main(String[] args) throws Exception {
    var builder = DistributedApplication.CreateBuilder(args);

    var web = builder.addExecutable(
        "web",
        "node",
        ".",
        new String[] { "service.mjs" });
    web.withHttpEndpoint(new WithHttpEndpointOptions().env("PORT"));
    web.withExternalHttpEndpoints();

    builder.build().run();
}
